bcgov/HOUS-permit-portal
# add editable nickname to permit applications and add submission infobox

## Description

add controller logic, migration, input and form, validation, etc for nickname
also add info box and readonly option for submitted forms

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-672
https://hous-bssb.atlassian.net/browse/BPHH-518

## Steps to QA

log in as submitter
view a draft permit, edit a nickname by clicking on, typing, clicking off
save and finish later - now goes back one to index after saving
go to a submitted application, see the blue info box
See fields are now disabled (readonly)

## UI Accessibility Checklist

passes wave
